### ISSUE TITLE
Allow adding --use-strict for 6to5-node as argument

### ISSUE THREAD

**Issue by terrybleger** [6to5/6to5#667]
In my use case, for node v0.11.x to be able to work without having any problem, I have to do this:

nodemon:
```
nodemon --use-strict --harmony --exec 6to5-node --experimental --playground --blacklist='useStrict,es6.constants,es6.blockScoping,regenerator,es6.forOf,es6.modules,es6.arrowFunctions' -- app.js
```

bash script:
```
exec /usr/local/bin/n as 0.11.16 /usr/local/bin/6to5-node --use-strict --harmony --blacklist='useStrict,es6.constants,es6.blockScoping,regenerator,es6.forOf,es6.modules,es6.arrowFunctions' --experimental --playground "$@"
``` 

The problem is, 6to5-node will reject --use-strict or any added --xxx.

The previous version of 6to5 has already included --harmony, but now is gone and must be explicitly set as argument. That's fine anyway, but please allow adding --use-strict for 6to5-node.

Thanks.

**Comment by sebmck** [comment]
Sure, should be easy enough to add it to the argument list [here](https://github.com/6to5/6to5/blob/master/bin/6to5-node#L29-39).

**Comment by sebmck** [comment]
Fixed as of 3.3.7, thanks!
